src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -2518,6 +2549,7 @@ void InitSolarHeatGains(EnergyPlusData &state)
 
             state.dataSurface->SurfSkySolarInc(SurfNum) = 0.0;
             state.dataSurface->SurfGndSolarInc(SurfNum) = 0.0;
+            state.dataSurface->GndReflSolarRad(SurfNum) = 0.0;
```

Comment:
A new surface variable `GndReflSolarRad` holds the ground reflected solar radiation for each exterior surface. Now the ground surface reflectance can be different for each exterior surface; hence, the global ground reflected solar radiation variable (`state.dataEnvrn->GndSolarRad`) cannot be used.